test/stats/cli-flags/stats.test.js

```diff
@@ -29,17 +29,10 @@ describe('stats flag', () => {
         }
     });
 
-    it('should warn when --verbose & --stats are passed together', () => {
-        const { stderr, stdout } = run(__dirname, ['--verbose', '--stats', 'normal']);
-        expect(stderr).toBeTruthy();
-        expect(stderr).toContain('Conflict between "verbose" and "stats" options');
-        expect(stdout).toBeTruthy();
-    });
-
     it('should warn when an unknown flag stats value is passed', () => {
         const { stderr, stdout } = run(__dirname, ['--stats', 'foo']);
         expect(stderr).toBeTruthy();
-        expect(stderr).toContain('invalid value for stats');
+        expect(stderr).toContain('WebpackOptionsValidationError');
```

Comment:
Can we improve message testing here and we can merge it :+1:

Reply:
done 👍